src/components/Calendar/doubleCalendar/index.js

```diff
@@ -0,0 +1,350 @@
+import React, { useMemo, useState, useCallback, useEffect } from 'react';
+import PropTypes from 'prop-types';
+import { useUniqueIdentifier } from '../../../libs/hooks';
+import {
+    UP_KEY,
+    DOWN_KEY,
+    RIGHT_KEY,
+    LEFT_KEY,
+    HOME_KEY,
+    END_KEY,
+    PAGEUP_KEY,
+    PAGEDN_KEY,
+    SPACE_KEY,
+    ENTER_KEY,
+} from '../../../libs/constants';
+import { Provider } from '../context';
+import RightIcon from '../icons/rightArrow';
+import LeftIcon from '../icons/leftArrow';
+import DaysOfWeek from '../daysOfWeek';
+import Month from '../month';
+import {
+    normalizeDate,
+    getFirstDayMonth,
+    addMonths,
+    getNextFocusedDate,
+    isSameMonth,
+    isEmptyRange,
+    isDateBelowLimit,
+} from '../helpers';
+import {
+    useYearsRange,
+    useLastDayMonth,
+    useMoveFocusedDayFunction,
+    useMoveFocusedMonthFunction,
+} from './hooks';
+import StyledControlsContainer from '../styled/controlsContainer';
+import StyledArrowButton from '../styled/arrowButton';
+import StyledMonthsContainer from './styled/monthsContainer';
+import StyledTable from '../styled/table';
+import StyledContainer from './styled/container';
+import StyledCalendar from './styled/calendar';
+import StyledDivider from './styled/divider';
```

Comment:
I think you should put an index.js in the styled folders and export all this styled component